app/forms/support/edit_course_form.rb

```diff
@@ -0,0 +1,80 @@
+module Support
+  class EditCourseForm
+    include ActiveModel::Model
+    include ActiveModel::Validations
+
+    attr_accessor :course_code, :name, :day, :month, :year
```

Comment:
Could we dine a const `FIELDS` for these like we do in the other forms. Then your `attr_accessor` could just be `attr_accessor(*FIELDS)` and your controller can permit the same fields